server/routes/userRouter.js

```diff
@@ -1,6 +1,7 @@
 const router = require("express").Router();
 
 const authController = require("../controllers/authController");
+const profileController = require("../controllers/profileControllers")
```

Comment:
Would probably be better to have a router for the profile outside of the user router